diff --git a/dist/index.js b/dist/index.js
index f5e8a86..342cb90 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -3996,8 +3996,9 @@ var coerce = {
 };
 var NEVER = INVALID;
 // src/types.ts
 var UserInputMessage = exports_external.object({
   type: exports_external.literal("user"),
+  steer: exports_external.literal(true).optional(),
   message: exports_external.object({
     role: exports_external.literal("user"),
     content: exports_external.array(exports_external.object({
